src/micromamba/devcontainer-feature.json

```diff
@@ -18,6 +18,11 @@
             "default": false,
             "description": "Reinstall in case Micromamba already exists"
         },
+        "autoActivate": {
+            "type": "boolean",
+            "default": true,
+            "description": "Auto activation of base environment"
+        },
```

Comment:
Could you add a new test for `autActivate: false`? (If it is automatically testable)

Reply:
I'd like to (and actually has already tried), but I did not figure out how the tests works.

Can you provide any suggestions on where should I start from?

Reply:
Write the installation options in a json file and test the expected commands in a shell script.

https://github.com/mamba-org/devcontainer-features/blob/400d53fe9b66ca793215c866021c245b90dcb51c/test/micromamba/scenarios.json#L13-L20

https://github.com/mamba-org/devcontainer-features/blob/400d53fe9b66ca793215c866021c245b90dcb51c/test/micromamba/install-specific-version.sh#L1-L13

Check the Dev Container CLI documentation for more information.
https://github.com/devcontainers/cli/blob/main/docs/features/test.md

Reply:
Thanks for your help. The test is now added.